src/main/java/adry/projectile/ProjectileFrame.java

```diff
@@ -0,0 +1,143 @@
+        add(angleLabel);
+        JLabel secondsLabel = new JLabel("Seconds");
+        add(secondsLabel);
+        JLabel blankSpace = new JLabel();
+        add(blankSpace);
+        JButton calculateButton = new JButton("Calculate");
+        add(calculateButton);
+        peakYlabel = new JLabel("Peak Y:");
+
+        // velocity
+        velocityField = new JTextField();
+        add(velocityField);
+
+        // JSlider to display angle from 0-90
+        final int min = 0;
+        final int max = 90;
+        final int initial = 45;
+        angleAdjuster = new JSlider(min, max, initial);
+        angleAdjuster.setMajorTickSpacing(15);
+        angleAdjuster.setPaintTicks(true);
+        angleAdjuster.setPaintLabels(true);
+        add(angleAdjuster);
+
+        // seconds
+        secondsField = new JTextField();
+        add(secondsField);
+
+        // x & y
+        JLabel labelX = new JLabel("X");
+        JLabel emptyX = new JLabel();
+        add(labelX);
+        add(emptyX);
+        JLabel labelY = new JLabel("Y");
+        JLabel emptyY = new JLabel();
+        add(labelY);
+        add(emptyY);
+        fieldX = new JTextField();
+        fieldY = new JTextField();
+
+        // add row that displays peak y
+        add(peakYlabel);
+        add(fieldY);
+
+        // add row that displays x intercept
+        add(interceptXlabel);
+        add(fieldX);
+
+        // add listeners
+        velocityField.getDocument().addDocumentListener(new SimpleDocumentListener() {
+            @Override
+            public void update(DocumentEvent e) {
+                calculate();
+            }
+        });
+
+        // slider
+        angleAdjuster.addChangeListener(new ChangeListener() {
+            @Override
+            public void stateChanged(ChangeEvent e) {
+                calculate();
+            }
+        });
+
+        // seconds
+        secondsField.getDocument().addDocumentListener(new SimpleDocumentListener() {
+            @Override
+            public void update(DocumentEvent e) {
+                calculate();
+            }
+        });
+
+        calculateButton.addActionListener(
+                new ActionListener() {
+                    public void actionPerformed(ActionEvent e) {
+                        Projectile projectile = new Projectile(
+                                Double.parseDouble(velocityField.getText()),
+                                angleAdjuster.getValue()
+                        );
+                        projectile.setSeconds(
+                                Double.parseDouble(secondsField.getText())
+                        );
+                        interceptXlabel.setText(Double.toString(projectile.getX()));
+                        peakYlabel.setText(Double.toString(projectile.getY()));
+                        fieldY.setText(Double.toString(projectile.getPeakY()));
+                        fieldX.setText(Double.toString(projectile.getInterceptX()));
```

Comment:
you should be calling calculate() like in the other Listeners

Reply:
should I switch to branch gui to commit this change or stay on graph?

Reply:
Did it on graph.

But merge the PR.